internal/controllers/core/portforward/reconciler.go

```diff
@@ -104,20 +107,15 @@ func (r *Reconciler) startPortForwardLoop(ctx context.Context, entry portForward
 
 	for {
 		start := time.Now()
-		err := r.onePortForward(ctx, entry, forward)
+		r.onePortForward(ctx, entry, forward)
 		if ctx.Err() != nil {
-			// If the context was canceled, we're satisfied.
-			// Ignore any errors.
+			// If the context was canceled, there's nothing more to do;
+			// we cannot even update the status because we no longer have
+			// a valid context, but that's fine because that means this
+			// PortForward is being deleted.
 			return
 		}
 
-		// Otherwise, repeat the loop, maybe logging the error
-		if err != nil {
-			logger.Get(ctx).Infof("Reconnecting... Error port-forwarding %s (%d -> %d): %v",
```

Comment:
hmm....it seems like we still want this logging to tell the user that we're reconnecting?

right now, it looks like we never print the error message at all....

Reply:
Yep, good catch. Re-added in 91a5eb6fa7ef9c74acf3bbe318014d29f3a72484.

FWIW I kept the logic the same in that it will log every invocation (that "maybe" in the comment was incorrect), but it might make sense to make this conditional along with API update to avoid flooding the logs (if you pick a privileged port it's...a lot)